workers/unity/Assets/Playground/Scripts/Workers/UnityClient.cs

```diff
@@ -45,6 +45,10 @@ public override void RegisterSystems()
 
             // Metric sending system
             World.GetOrCreateManager<MetricSendSystem>();
+
+            // Score update systems
+            World.GetOrCreateManager<ProcessScoresSystem>();
```

Comment:
Why is this on both the client and the game logic workers? Only one of them will receive the command (the game logic since its authoritative over the `Launchable` component). 

The pattern should be closer to: 
Collision happens -> send command to increment score -> command received by the worker authoritative over the launchable component, update the score on the component -> client listens for updates and updates the score UI locally.